### DISCUSSION TITLE
Tutorial 3 (Gromacs)

### DISCUSSION THREAD

**Discussion by shvel** [SIRAHFF/documentation#84] [General]
Hi Guys

I am doing the tutorial 3 (protein) in gromacs... But I do not understand why it is necessary the index file during the production run...  =>  -n ../1CRN_cg_ion.ndx
I though it was only necessary for the equilibration stages (restriction positions)...

Besides, when I try to simulate several proteins, I found a error with the position restrains directive in topol. 
could you give me an idea of how to generate the itp files, i.e, do i have to create the itp files ( bkbres.itp & bkres_soft.itp) for each protein?

Thanks a lot in advance

**Comment by SIRAHFF** [comment]
Hi! Sorry about the late answer.

Indeed, the -n flag is not necessary.

Regarding the error. It is a gromacs-related thing. If you are generating restraint files for chains with different lengths, you need to be sure the corresponding .itp files have the correct number of lines. You have to create a group for each chain and, from there, an index file. With that, you can create restraint files for each chain using gmx genrestr.
> gmx genrestr -f ThePDBOfYourChain.pdb -n IndexOfYourPDB.ndx -o YourRestraintFile.itp

Best,
The SIRAH Team
